src/components/app/data/hooks/usePassLearnerCsodParams.js

```diff
@@ -1,33 +1,40 @@
 import { useEffect } from 'react';
+import { logError } from '@edx/frontend-platform/logging';
 import { useLocation, useParams } from 'react-router-dom';
 import { updateUserCsodParams } from '../services';
+import useEnterpriseCustomer from './useEnterpriseCustomer';
 
-const usePassLearnerCsodParams = (enterpriseUUID) => {
+const usePassLearnerCsodParams = () => {
   const location = useLocation();
   const { courseKey } = useParams();
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
 
-  useEffect(async () => {
+  useEffect(() => {
```

Comment:
The callback function passed to `useEffect` cannot be `async`. The prior implementation (just recently merged...) was failing `npm run lint` in CI due to the `async`. Refactored to wrap the API call in a `async` helper function instead.

Also, refactored to early return instead when the necessary query params are not included.